server/graphql/programs/src/queries/program.rs

```diff
@@ -22,7 +26,7 @@ pub fn programs(
     let user = validate_auth(
         ctx,
         &ResourceAccessRequest {
-            resource: Resource::QueryProgram,
+            resource: Resource::QueryMasterList,
```

Comment:
This is used for both Patient & Requisition programs... and doesn't make sense to use QueryProgram permission when that is a Patient Program specific permission... though it has a lot of other checks... maybe should have just created a dup with query masterlist permission? 👀